telebot/types.py

```diff
@@ -97,12 +100,32 @@ def check_json(json_type, dict_copy = True):
         else:
             raise ValueError("json_type should be a json dict or string.")
 
-    def __str__(self):
-        d = {
-            x: y.__dict__ if hasattr(y, '__dict__') else y
-            for x, y in self.__dict__.items()
-        }
-        return str(d)
+    def __str__(self) -> str:
+        default = (
```

Comment:
Frankly speaking, I do not understand anything in this pigeon talk. It takes too much time to decode.

Can this code be rewriten in adequate if / then / else mode? So ANYBODY who see it can understand what will be returned in what case?

Reply:
OK